pre-lollipop-activity-transition/src/main/java/com/kogitune/activity_transition/SimpleAnimatorListener.java

```diff
@@ -0,0 +1,28 @@
+package com.kogitune.activity_transition;
+
+import android.animation.Animator;
+
+/**
+ * Created by Joe on 2016/10/8.
+ * Email lovejjfg@gmail.com
+ */
+
+public abstract class SimpleAnimatorListener implements Animator.AnimatorListener {
```

Comment:
Probably you can use AnimatorListenerAdapter.
https://developer.android.com/reference/android/animation/AnimatorListenerAdapter.html